### ISSUE TITLE
whitespace_011 does not work when using attributes

### ISSUE THREAD

**Issue by GCHQDeveloper211** [jeremiah-c-leary/vhdl-style-guide#1415]
**Environment**
VHDL Style Guide (VSG) version: 3.30.0 - Rocky 9

**Describe the bug**
Spaces are not inserted when values are attributes.
In the whitespace_011 rule, if you have an arithmetic operator like `foo'LEFT+1` it will not fix it and instead skip over.

**To Reproduce**
an example like: `read_host'LEFT+type_bits` run vsg with whitespace_011 enabled. This will not have spaces inserted 

**Expected behavior**
The above example should look like `read_host'LEFT + type_bits` after linting

**Screenshots**

**Additional context**

**Comment by GCHQDeveloper211** [comment]
Hi @jeremiah-c-leary,

I have cloned your repo and checked out the branch and executed vsg against what is in the ./bin directory. Unfortunately, it is not detecting an issue when using attributes still. But it does detect it if I remove the attribute. 

Thanks,
GCHQDeveloper211

**Comment by jeremiah-c-leary** [comment]
Evening @GCHQDeveloper211 ,

Is it the same issue as you reported with the `read_host'left+1`, or is it something new?

I added some examples to the test input file `<working copy>/tests/whitespace/rule_011_test_input.vhd` starting at line 137.  The fixed version is located at `<working copy>/tests/whitespace/rule_011_test_input.fixed.vhd`.

Could you run the unittest against that file:  `python -m unittest tests.whitespace.test_rule_011`.

If that passes, could you try running VSG:  `bin/vsg -f tests/whitespace/rule_011_test_input.vhd --fix` and see if whitespace is added to line 144 through 148?

Thanks,

--Jeremy

**Comment by GCHQDeveloper211** [comment]
Hi Jeremy,

I ran your tests and they passed ok.
Although when I ran your recommended command, it seems whitespace was not added to the `+` and `-` operators, but it was added to the `\` `*` and `**` operators.

<img width="269" alt="Image" src="https://github.com/user-attachments/assets/aa864585-bca0-4322-bdb8-57b1ca92c6ac" />

**Comment by jeremiah-c-leary** [comment]
Morning @GCHQDeveloper211 ,

The original issue was due to the `+` and `-` being classified as sign instead of a math operator.  If you run this command:
```mono
bin/vsg_parser -f tests/whitespace/rule_011_test_input.vhd
```
and find where line 144 is displayed:

![Image](https://github.com/user-attachments/assets/d20d95a3-7ade-4e55-8623-cb4bb24fcbdc)

You can see the `+` operator is classified as a `adding_operator.plus`, which is correct.

Previously it was being classified as a sign operator:

![Image](https://github.com/user-attachments/assets/e33cad8e-6124-4da5-8312-be60ed2c1de1)

I used this command to generate the following diff:
```mono
cp tests/whitespace/rule_011_test_input.vhd rule_011_test_input.fixed.vhd ;bin/vsg -f rule_011_test_input.fixed.vhd --fix
```

![Image](https://github.com/user-attachments/assets/be1d6b89-104e-4377-a124-f4a03954acd1)

This is my git status:

![Image](https://github.com/user-attachments/assets/95d1ecd2-6330-45c6-9103-ab305e3e58be)

I suspect the updates I made are not being executed for some reason.  Almost as if the branch VSG is not being used.

Regards,

--Jeremy

**Comment by jeremiah-c-leary** [comment]
Evening @GCHQDeveloper211 ,

There is another user with an issue similar to yours where an update on a branch was not being executed in issue #1403 .  Assuming you have VSG installed via PIP, could you try uninstalling VSG and then trying to run on the branch?

Thanks,

--Jeremy

**Comment by GCHQDeveloper211** [comment]
Hi I have managed to test this in a new environment, and yes this fixes the issue!

**Comment by jeremiah-c-leary** [comment]
Morning @GCHQDeveloper211 ,

That is awesome.  I will get this merged to master.

--Jeremy
